recipes/wise_enum/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class WiseEnumConan(ConanFile):
```

Comment:
Needed for my suggestion in the source method.
```suggestion
import os

required_conan_version = ">=1.33.0"

class WiseEnumConan(ConanFile):
```